docs/self-hosted-appcircle/install-server/helm-chart/configuration/advanced-configuration.md

```diff
@@ -125,523 +125,157 @@ webhook:
-| resourcesPreset                                   | string               | none          | Set container resources according to one common preset (allowed values: none, nano, small, medium, large, xlarge, 2xlarge). This is ignored if 'resources' is set ('resources' is recommended for production). |
-| schedulerName                                     | string               |               | Name of the k8s scheduler (other than default) for pods                                                                                                                                                        |
-| terminationGracePeriodSeconds                     | string               | -             | Seconds pods need to terminate gracefully                                                                                                                                                                      |
-| tolerations                                       | array                | []            | Tolerations for pods assignment                                                                                                                                                                                |
-| topologySpreadConstraints                         | array                | []            | Topology Spread Constraints for %%MAIN_CONTAINER_NAME%% pod assignment spread across your cluster among failure-domains                                                                                        |
-| updateStrategy                                    | object               | -             | Deployment strategy type                                                                                                                                                                                       |
-| updateStrategy.type                               | string               | RollingUpdate | Can be set to RollingUpdate or Recreate                                                                                                                                                                        |
-| volumePermissions                                 | ['object', 'string'] | -             | 'volumePermissions' init container parameters                                                                                                                                                                  |
+| Parameter                                                     | Description                                                               | Default Value                 |
+| ------------------------------------------------------------- | ------------------------------------------------------------------------- | ----------------------------- |
```

Comment:
- [ ] Review descriptions for docs convention
- [ ] Check default values | correctness?